rbd.c

```diff
@@ -144,6 +141,9 @@ static int tcmu_rbd_open(struct tcmu_device *dev)
 		goto rbd_close;
 	}
 
+	tcmu_dbg("config %s, size %lld\n", tcmu_get_dev_cfgstring(dev),
+			rbd_size);
+
```

Comment:
I fixed the tabbing/spacing here, so the rbd_size is aligned with the opening "(", so it consistent with "most" (I know in a lot of places it is a mismatch) of the other coding style we are using.